pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/ModelParser.groovy

```diff
@@ -128,6 +129,17 @@ class ModelParser implements Parser {
 
         ModelASTPipelineDef r = new ModelASTPipelineDef(pst);
 
+        List<ImportNode> additionalImports = []
+        [src.imports, src.staticImports, src.starImports, src.staticStarImports].each { imports ->
+            imports.each { ImportNode i ->
+                additionalImports.add(i)
+            }
+        }
+
+        if (!additionalImports.isEmpty()) {
+            r.setAdditionalImports(additionalImports.collect { it.text })
```

Comment:
Why not just skip the extra iteration and add `i.text` directly to `additionalImports`?